openapi/build.rs

```diff
@@ -14,4 +14,9 @@ fn main() {
     }
 
     println!("cargo:rerun-if-changed=../nix/pkgs/openapi-generator");
+    println!("cargo:rerun-if-changed=../control-plane/rest/openapi-specs");
+    // seems the internal timestamp is taken before build.rs runs, so we can't set this
+    // directive against files created during the build of build.rs??
+    // https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed
+    // println!("cargo:rerun-if-changed=.");
```

Comment:
Can this line be deleted?

Reply:
I thought of keeping it for reference and in case the cargo timestamp changes